AzureAppServiceSetAppSettings/Tasks/ConnectionStrings/ApplyConnectionStrings.ps1

```diff
@@ -10,7 +10,7 @@ param(
     $ResourceGroupName,
 
     [String] [Parameter(Mandatory = $false)]
-    $Slot,
+    $Slot = "",
```

Comment:
Isn't it better to use "production" as the default value of the $Slot parameter? Then you won't need the check and reassignment at line 25-27.

Reply:
I'd second that!